app/scripts/controllers/authentication/auth-snap-requests.ts

```diff
@@ -0,0 +1,37 @@
+import type { SnapId } from '@metamask/snaps-sdk';
+import type { HandleSnapRequest } from '@metamask/snaps-controllers';
+import { HandlerType } from '@metamask/snaps-utils';
+
+type SnapRPCRequest = Parameters<HandleSnapRequest['handler']>[0];
+
+const snapId = 'npm:@metamask/message-signing-snap' as SnapId;
+
+export function createSnapPublicKeyRequest(): SnapRPCRequest {
```

Comment:
Nit: Seems a bit unnecessary to have utility functions for these 😅

Reply:
Overkill yes. But these will be reused in future controllers 😃 (future PRs available soon).